src/components/LinkButton/NeutralLinkButton/NeutralLinkButton.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FC, Ref } from 'react';
+import {
+  BaseLinkButton,
+  LinkButtonIconAlign,
+  LinkButtonProps,
+  LinkButtonShape,
+  LinkButtonSize,
+  LinkButtonTextAlign,
+  LinkButtonType,
+} from '..';
+import { mergeClasses } from '../../../shared/utilities';
+
+import styles from '../linkbutton.module.scss';
+
+export const NeutralLinkButton: FC<LinkButtonProps> = React.forwardRef(
+  (
+    {
+      alignIcon = LinkButtonIconAlign.Left,
+      alignText = LinkButtonTextAlign.Center,
+      allowDisabledFocus = false,
+      ariaLabel,
+      classNames,
+      counter,
+      disabled = false,
+      dropShadow = false,
+      floatingLinkButtonProps,
+      iconProps,
+      linkButtonWidth,
+      nudgeProps,
+      onClick,
+      role,
+      shape = LinkButtonShape.Pill,
+      size = LinkButtonSize.Medium,
+      style,
+      target,
+      text,
+      ...rest
+    },
+    ref: Ref<HTMLAnchorElement>
+  ) => {
+    const linkButtonClassNames: string = mergeClasses([
+      classNames,
+      styles.linkButton,
+      styles.linkButtonNeutral,
+    ]);
+
+    return (
+      <BaseLinkButton
```

Comment:
Can we just expose BaseLinkButton as LinkButton, and let devs specify the type to get the style they want?

Reply:
Isn't it easier to lookup references on imports of distinct buttons?

Reply:
I suppose I could make this change. though deprecating the other buttons will take quite a long time.